ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -800,3 +800,97 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS4.3
   contributors: lucasmrod
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Password Account Lockout Threshold Is Configured (Fleetd required)
+  platforms: macOS
+  platform: darwin
+  description: |
+    The account lockout threshold specifies the amount of times a user can enter an incorrect password before a lockout will occur.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that Ensure Password Account Lockout Threshold.
+    Profile Method:
+    Create or edit a configuration profile with the following information:
+      1. The PayloadType string is com.apple.mobiledevice.passwordpolicy 
+      2. The key to include is maxFailedAttempts
+      3. The key must be set to <integer><value≤5></integer>
+  query: SELECT 1 FROM  pwd_policy where maxFailedAttempts <= 5;
```

Comment:
The PDF has the following note (to prefer using profiles instead of the command, i.e. `managed_policies` on our case):
```
Note: The profile method is the preferred method for setting password policy since - setglobalpolicy in pwpolicy is deprecated and will likely be removed in a future macOS release.
```

Given that you are using the profiles to set the configuration then it makes sense to rely on `managed_policies` instead of creating a new table for getting the password policies.

But given that the table is already implemented I'm ok to leave as is and improve later if need be.